backend/app/crud/prompts.py

```
@@ -0,0 +1,191 @@
+        .order_by(Prompt.updated_at.desc())
+        .offset(skip)
+        .limit(limit)
+    )
+    return session.exec(stmt).all()
+
+
+def count_prompts_in_project(session: Session, project_id: int) -> int:
+    return session.exec(
+        select(func.count()).select_from(
+            select(Prompt)
+            .where(Prompt.project_id == project_id, Prompt.is_deleted == False)
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not Prompt.is_deleted:` for false checks

Replace with `not Prompt.is_deleted`

(E712)